addons/mail/static/src/models/composer_view.js

```diff
@@ -210,6 +210,14 @@ registerModel({
             }
             this.discard();
         },
+        /**
+         * Called when clicking on "discard" button.
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickDiscard(ev) {
+            this.discard();
+        },
         /**
          * @private
```

Comment:
Not related to this PR: it appears that a few handlers in this file are incorrectly documented as private.